sentry_sdk/integrations/asgi.py

```diff
@@ -163,8 +163,8 @@ async def _run_app(self, scope, receive, send, asgi_version):
         _asgi_middleware_applied.set(True)
         try:
             hub = Hub(Hub.current)
-            with auto_session_tracking(hub, session_mode="request"):
-                with hub:
+            with hub:
```

Comment:
Do we need this hub stuff here at all still?

Reply:
we do not need it here. but we can have it here. plan is to first make everything work with the code as is (so we do not break users code) and in later minor release upgrade all our integrations to use the new API